src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -194,12 +192,7 @@ public void BuildBasicApplicationReleaseProfiledAotWithoutDefaultProfile ()
 						proj.OutputPath, $"{proj.PackageName}-Signed.apk");
 
 					var helper = new ArchiveAssemblyHelper (apk, usesAssemblyBlobs);
-					if (!Builder.UseDotNet) {
-						// BundleAssemblies=true
-						Assert.IsFalse (helper.Exists ("assemblies/UnnamedProject.dll"), $"UnnamedProject.dll should not be in the {proj.PackageName}-Signed.apk");
-					} else {
-						Assert.IsTrue (helper.Exists ("assemblies/UnnamedProject.dll"), $"UnnamedProject.dll should be in the {proj.PackageName}-Signed.apk");
-					}
+					Assert.IsTrue (helper.Exists ("assemblies/UnnamedProject.dll"), $"UnnamedProject.dll should be in the {proj.PackageName}-Signed.apk");
```

Comment:
Shouldn't this assert be conditioned on `usesAssemblyBlobs`?

Reply:
The helper hides that from the tests.